lib/kb_ea_utils/kb_ea_utilsImpl.py

```diff
@@ -209,7 +209,7 @@ def run_app_fastq_ea_utils_stats(self, ctx, input_params):
         info = None
         readLibrary = None
         try:
-            readLibrary = wsClient.get_objects([{'ref': input_reads_ref}])[0]
+            readLibrary = wsClient.get_objects2({'objects':[{'ref': input_reads_ref}]})['data'][0]
```

Comment:
Is this an update to use new functionality in GO2? If so, a test should be added to exercise that functionality such that if the call were switched back to GO1 the test would fail.